server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -1378,7 +1378,7 @@ public boolean isMajorCompactionQueued() {
   }
 
   /**
-   * Interrupts and waits for any running compactions. After this method returns no compactions
+   * Interrupts and waits for any running compactions. After this method returns, no compactions
```

Comment:
This is the only change that seems to improve understand-ability but thought I would include the other typos while the file was being changed anyways.